### DISCUSSION TITLE
Automerge per repo?

### DISCUSSION THREAD

**Discussion by dbimhoff** [renovatebot/renovate#10614] [Request Help] [answered]
Hello, I'm trying to test out automerging, but want to try it first on a single repo and see if I like it then integrate it into the rest of my repos. I was looking for a configuration to do just a single repo for auto-merging, but I couldn't find anything. Does that exsist?

**Comment by rarkins** [comment, marked as answer]
If you want settings to be per-repo, add them to the in-repo "renovate.json" file. Nearly all Renovate capabilities are intended to be configured that way.

**Comment by HonkingGoose** [comment]
@dbimhoff You might like to read these guides/docs from us:

- [Noise Reduction](https://docs.renovatebot.com/noise-reduction/)
- [Automerging Updates](https://docs.renovatebot.com/automerge-configuration/)
